README.md

```diff
@@ -162,225 +189,414 @@ and modify the `system-reserved` line under `kubeletExtraArgs`:
+
+    4. Click on the **Create application** button.
+
+    5. Enter an application name (we'll use the name `my-app-one` in
+    this documentation, but you can use any name you prefer).
+
+    6. Click the **Create application** button.
+
+    7. On the `my-app-one` application screen, click on the **Components** tab.
+
+    8. Click on the **Add component** button.
+
+    9. In a separate browser tab, go to the Git repository you
+    forked from `https://github.com/konflux-ci/testrepo`.
+
+    10. Click on the **Code** button.
+
+    11. Click on HTTPS.
+
+    12. Copy the git URL (it should end in `.git`).
+
+    13. Go back to your Konflux browser tab and paste the Git
+    URL into the **Get repository URL** field.
+
+    14. Change the component name in the **Component name**
+    field if you wish, or you can accept the default name.
+
+    15. Set the **Pipeline** drop-down list to `docker-build-oci-ta`
+    if it isn't already.
+
+    16. Leave all other fields blank or set to their default settings.
+
+    17. Click the **Add component** button.
+
+    **Note**: If you encounter 404 Not Found error, refer to the
+    [troubleshooting guide](https://github.com/konflux-ci/konflux-ci/blob/main/docs/troubleshooting.md#unable-to-create-application-with-component-using-the-konflux-ui).
+
+Now, if you click on the **Components** tab in the Konflux UI, you
+should see the component you just created.
+
+**Note**: If you have *NOT* completed the [Quay.io](https://quay.io) setup
+steps in the previous section, Konflux will be *UNABLE* to send pull requests
+to your repository.
+
+In your GitHub repository, you should now see that Konflux created a pull
+request with two new pipelines. Konflux triggers one pipeline when pull request
+events occur, such as when you create or modify pull requests, and another
+when push events occur, such as when you merge a pull request.
+
+You have successfully onboarded your application, and you can continue to the
 [next step](#observe-the-behavior).
 
 ### Option 2: Onboard Application with Kubernetes Manifests
 
-With this approach, we use `kubectl` to deploy the manifests for creating the
-`Application` and `Component` resources and we manually create the PR for introducing
-the pipelines to run using Konflux.
+With this approach, we use `kubectl` to deploy the manifests for creating
+the Application and Component resources, and we manually create the pull
+request to configure and run the pipelines using Konflux.
 
-To do that:
+Let's step through the process of deploying a manifest via the terminal.
 
-1. :gear: Use a text editor to edit your local copy of the
-   [example application manifests](./test/resources/demo-users/user/ns2/application-and-component.yaml):
+1. :gear: Use your text editor of choice to edit your local copy of the
+[example application manifests](https://github.com/konflux-ci/konflux-ci/blob/main/test/resources/demo-users/user/ns2/application-and-component.yaml).
```

Comment:
I think that in this case, contrary to the current and suggested text, it will make sense to have the path to the file visible directly, as this is what the user will need in order to locate the file on their local repo.

```suggestion
1. :gear: Use your text editor of choice to edit your local copy of the
example application manifests at
`test/resources/demo-users/user/ns2/application-and-component.yaml`.
```